content/blog/2022/07/camel-quarkus-release-2.11.0/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Camel Quarkus 2.11.0 Released"
+date: 2022-07-27
+authors: ["jamesnetherton"]
+categories: ["Releases", "Camel Quarkus"]
+preview: "Camel Quarkus 2.11.0 Released"
+summary: "Camel Quarkus 2.11.0 is released and aligns with Camel 3.18.0 and Quarkus 2.11.0.Final"
+---
+
+<sub><sup>Image by <a href="https://www.flickr.com/photos/lanzen/5984113332">Anders Lanzen</a> <a href="https://creativecommons.org/licenses/by-nc-sa/2.0">CC BY-NC-SA 2.0</a></sup></sub>
+
+We are pleased to announce the 2.11.0 release of Camel Quarkus.
+It aligns with Camel 3.18.0 and Quarkus 2.11.0.Final.
```

Comment:
seems the Quarkus version that will be released is 2.11.1.Final due to last minute security fix

Reply:
So we have to upgarde to 2.11.1.Final in camel-quarkus?

Reply:
We can upgrade on our `main` branch. But when the quarkus-platform is released, it will enforce 2.11.1.Final anyway.

Reply:
I updated the versions to reflect the new Quarkus 2.11 release.